channels/rdpgfx/client/rdpgfx_main.c

```diff
@@ -1716,11 +1724,8 @@ UINT DVCPluginEntry(IDRDYNVC_ENTRY_POINTS* pEntryPoints)
 		if (gfx->H264)
 			gfx->SmallCache = TRUE;
 
-		if (gfx->SmallCache)
-			gfx->ThinClient = FALSE;
-
-		gfx->MaxCacheSlot = (gfx->ThinClient) ? 4096 : 25600;
-		context = (RdpgfxClientContext*) calloc(1, sizeof(RdpgfxClientContext));
+		gfx->MaxCacheSlot = gfx->SmallCache ? 4096 : 25600;
```

Comment:
Your comment above implies that the cache is also small if RDPGFX_CAPS_FLAG_THINCLIENT is set, correct?

Reply:
Yes, that's what the spec says. thinclient implies the use of remoteFx (instead of progressive) and the cache to fit in 16Mb (so a small cache).

Reply:
Just asking, because here you only check ```gfx->SmallCache``` and not ```gfx->ThinClient```